src/elm/Pages/Build/View.elm

```diff
@@ -14,6 +14,7 @@ module Pages.Build.View exposing
 import Ansi.Log
 import Array
 import DateFormat.Relative exposing (relativeTime)
+import Debug exposing (toString)
```

Comment:
the [Debug](https://package.elm-lang.org/packages/elm/core/latest/Debug) package is not available for use in production. 

is there an alternative to `toString` we can use? 
what is this function accomplishing?